SPECS-EXTENDED/ruby-augeas/ruby-augeas.spec

```diff
@@ -0,0 +1,176 @@
+Name:           ruby-augeas
+Version:        0.5.0
+Release:        29%{?dist}
+Summary:        Ruby bindings for Augeas
+Vendor:		Microsoft Corporation
+Distribution:	Mariner
+License:        LGPLv2+
+URL:            http://augeas.net
```

Comment:
Nit: let's switch to HTTPS, since it works.

Reply:
fixed